src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -174,45 +174,49 @@ Pooling::Pooling(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr c
         }
     };
 
-    if (auto maxPoolOp_v8 = ov::as_type_ptr<const ov::op::v8::MaxPool>(op)) {
-        isMaxPool8 = true;
+    if (auto maxPoolOpBase = ov::as_type_ptr<const ov::op::util::MaxPoolBase>(op)) {
         algorithm = Algorithm::PoolingMax;
         poolingAttrs.exclude_pad = false;
-        poolingAttrs.rounding = maxPoolOp_v8->get_rounding_type();
-        poolingAttrs.pad_type = maxPoolOp_v8->get_auto_pad();
+        poolingAttrs.rounding = maxPoolOpBase->get_rounding_type();
+        poolingAttrs.pad_type = maxPoolOpBase->get_auto_pad();
+        get_attributes(poolingAttrs.stride, maxPoolOpBase->get_strides());
+        get_attributes(poolingAttrs.kernel, maxPoolOpBase->get_kernel());
+        get_attributes(poolingAttrs.data_pad_begin, maxPoolOpBase->get_pads_begin());
+        get_attributes(poolingAttrs.data_pad_end, maxPoolOpBase->get_pads_end());
+        poolingAttrs.auto_pad = (poolingAttrs.pad_type == ov::op::PadType::SAME_LOWER || poolingAttrs.pad_type == ov::op::PadType::SAME_UPPER);
+    }
 
+    if (auto maxPoolOp_v14 = ov::as_type_ptr<const ov::op::v14::MaxPool>(op)) {
+        get_attributes(poolingAttrs.dilation, maxPoolOp_v14->get_dilations());
+    } else if (auto maxPoolOp_v8 = ov::as_type_ptr<const ov::op::v8::MaxPool>(op)) {
+        isMaxPool8 = true;
```

Comment:
The `isMaxPool8` variable is a condition for the further logic for the second output. It should be reflected or resolved for `v14::MaxPool` as well.
https://github.com/openvinotoolkit/openvino/blob/05af9b9a7e5cc3e3a1c5c53b8b72dd1cf5947e09/src/plugins/intel_cpu/src/nodes/pooling.cpp#L649-L656